### DISCUSSION TITLE
No space left on device\nerror: unable to write file...

### DISCUSSION THREAD

**Discussion by rrueth** [renovatebot/renovate#16214] [Request Help]
I'm trying to set up Renovate on a relatively large, private repo. It is about 8GB fully cloned. Looking at the Renovate dashboard, it appears that Renovate is failing when trying to clone the repo and runs out of space. I really only need Renovate to see and update one file in the repo.

Is there any way for me to specify only that one file? Or is there a way that I can exclude the parts of the repository that include the majority of large files?

The ID of the most recently failing run is 712867454.

Thanks in advance!

**Comment by rarkins** [comment]
Renovate currently does a blobless clone, which means that full 8GB would be pulled down every time. Related feature request is sparse checkout: #7840

**Reply by rrueth** [reply]
@rarkins, thank you for the quick response! Beyond adding my +1 to #7840, is there anything that I can do right now to enable Renovate for our 8GB repository (outside of making the repo smaller =)?
